jib/jib-gradle.yaml

```diff
@@ -14,3 +17,4 @@ spec:
     - jib
     - -Duser.home=/builder/home
     - --image=${IMAGE}
+    workingDir: /workspace/${DIRECTORY}
```

Comment:
This can probably just be `${DIRECTORY}`, the build system will interpret relative paths assuming you mean under `/workspace`

Reply:
Oh, that's cool, didn't know it infers that.

Reply:
Hmm, okay. I bet we're not handling that correctly in the build controller. Let's stick with this and I'll make a note to investigate more later.